src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -71,4 +87,22 @@ public void toStringMethod() {
                 + editPersonDescriptor.getTags().orElse(null) + "}";
         assertEquals(expected, editPersonDescriptor.toString());
     }
+
+    @Test
+    public void fromPerson() {
+        EditPersonDescriptor expected = new EditPersonDescriptor();
+        expected.setId(new Id(VALID_ID_BOB));
+        expected.setName(new Name(VALID_NAME_BOB));
+        expected.setMajor(new Major(VALID_MAJOR_BOB));
+        expected.setAddress(new Address(VALID_ADDRESS_BOB));
+        expected.setIntake(new Intake(VALID_INTAKE_BOB));
+        expected.setPhone(new Phone(VALID_PHONE_BOB));
+        expected.setEmail(new Email(VALID_EMAIL_BOB));
+        Set<Tag> tags = new HashSet<>();
+        tags.add(new Tag(VALID_TAG_FRIEND));
```

Comment:
Consider modifying tags so that they are more relevant to out project. However, this can be done in future iterations.